ruby_on_rails/aws.md

```diff
@@ -15,17 +15,22 @@ but for that, we most often use Google Firebase.
 
 ### Preconditions
 
+#### renuo-cli
+
 You will need Renuo-CLI to be set up and at the newest version:
 
 `gem install renuo-cli` --> see [renuo-cli](https://github.com/renuo/renuo-cli)
 
 Make sure `renuo -v` shows the [newest version](https://github.com/renuo/renuo-cli/tags)
 
+#### aws-cli
+
+You'll need to install `aws-cli`. No worries, if you don't have it alreay. The setup command below will guide you.
+It will install `aws-cli` and ask you for the `renuo-app-setup` credentials you find in our password manager.
```

Comment:
```suggestion
It will install `aws-cli` and ask you for the `renuo-app-setup` credentials, which you may find in our password manager. If not please ask `wg-operations` for help.
```